mikrotik/resource_dns_record_test.go

```diff
@@ -142,9 +178,9 @@ func TestAccMikrotikDnsRecord_import(t *testing.T) {
 					resource.TestCheckResourceAttrSet(resourceName, "id")),
 			},
 			{
-				ImportState:       true,
-				ResourceName:      resourceName,
-				ImportStateId:     dnsName,
+				ImportState:  true,
+				ResourceName: resourceName,
+				// ImportStateId:     dnsName,
```

Comment:
Any reason for keeping this around?

Reply:
no reason, overlooked that 
eagle eye 🦅